<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>简单ComboBox</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="btnIniItems">
          <property name="text">
           <string>初始化列表</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btnClearItems">
          <property name="text">
           <string>清除列表</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QCheckBox" name="chkBoxEditable">
          <property name="text">
           <string>可编辑</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>北京市</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/icons/UNIT.ICO</normaloff>:/images/icons/UNIT.ICO</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>上海市</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/icons/UNIT.ICO</normaloff>:/images/icons/UNIT.ICO</iconset>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>有用户数据的ComboBox</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnIni2">
          <property name="text">
           <string>初始化城市+区号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox2"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string>QPlainTextEdit（具有标准快捷菜单）</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnToComboBox">
          <property name="text">
           <string>文本框内容添加到ComboBox</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClearText">
          <property name="text">
           <string>清除文本内容</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkBoxReadonly">
          <property name="text">
           <string>Read Only</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
